dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp

```diff
@@ -120,12 +120,11 @@ ModelBusContainer::exploreNeighbors() {
   subNetworks_.erase(subNetworks_.end() - 1);
 
   Trace::debug() << DYNLog(NbSubNetwork, numSubNetwork, subNetworks_.size()) << Trace::endline;
-  Trace::debug("NETWORK") << DYNLog(NbSubNetwork, numSubNetwork, subNetworks_.size()) << Trace::endline;
+  Trace::debug(Trace::NETWORK) << DYNLog(NbSubNetwork, numSubNetwork, subNetworks_.size()) << Trace::endline;
```

Comment:
I don't understand why we have this one twice (in the main log file and in the Network log file). I would keep it only in the network log file and will simplify it because I have the impression that we give twice the same information (numSubNetwork and subNetworks_.size()).

Reply:
Done